src/main/java/com/pjait/byt/pettracker/Person.java

```diff
@@ -3,18 +3,15 @@
 import lombok.Getter;
 import lombok.Setter;
 
-import java.sql.Date;
-import java.sql.Timestamp;
-
 @Getter
 @Setter
-public class Person {
+public abstract class Person {
```

Comment:
update the Person class constructor as mentioned in the review comment and accordingly, you can update the User class constructor.